mshadow/random.h

```diff
@@ -152,24 +272,39 @@ class Random<cpu, DType> {
   std::mt19937 rnd_engine_;
   /*! \brief random number seed used in random engine */
   unsigned rseed_;
-  // implementing generators.
-  inline void GenUniform(DType *dptr, index_t size, DType a, DType b) {
```

Comment:
Should say: All distributions sampled on the CPU use SampleDistribution. GPU functionality is fully unchanged and still restricted to uniform/gaussian.